snapcraft.yaml

```diff
@@ -650,6 +650,30 @@ parts:
       - bin/nvidia-container-cli*
       - lib/libnvidia-container*.so*
 
+  nvidia-container-toolkit:
```

Comment:
This is weird because I built the snap like this and it did not cause any issue even without a `build-packages: go`.. And yes the compiler to use is Go.

Reply:
Its probably relying on an earlier part that installed Go - which works but is brittle.

Reply:
I agree that we should explicitly depend on Go, however I think we want:

```
    build-snaps:
      - go
```

Reply:
fixed